input/pagecontent/index.md

```diff
@@ -125,8 +125,8 @@ This guide is divided into several pages which are listed at the top of each pag
   - [Profiles and Extensions](profiles-and-extensions.html): This page describes the profiles and extensions that are defined in this guide to exchange quality data. Each profile page includes a narrative description and guidance, formal definition and a "Quick Start" guide which summarises the supported search transactions for each profile. Although the guidance typically focuses on the profiled elements, it may also may focus on un-profiled elements to aid with implementation.
```

Comment:
For AUCDI line replace with:
This page maps AUCDI data groups and elements to FHIR artefacts in AU Core.

Reply:
Fixed, thanks.

Reply:
1. Fixed
2. Can't see the change.

Reply:
Sorry, fixed.

Reply:
This change for number 2, overwrote the change to number 1 and has been was applied to the wrong line. Put back the line for AU Core Data for Interoperability and move this new text to the line for Relationship with Other IGs.

Reply:
Fixed, sorry.